db/migrate/20161125161211_add_all_day_to_holidays.rb

```diff
@@ -0,0 +1,10 @@
+class AddAllDayToHolidays < ActiveRecord::Migration[5.0]
+  def up
+    add_column :holidays, :all_day, :boolean, default: false, null: false
+    execute 'UPDATE holidays SET all_day = true WHERE bank_holiday = true'
```

Comment:
I know it's equivalent but I see no good reason for explicit SQL when `Holiday.where(all_day: true).update_all(bank_holiday: true)` infers for you. We should avoid SQL unless absolutely necessary.